app/core/views/_components/html.pug

```diff
@@ -9,9 +9,12 @@ mixin head()
     meta(http-equiv='X-UA-Compatible', content='IE=Edge')
     meta(content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name='viewport')
     link(href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet')
+    link(href='/css/home.min.css', rel='stylesheet')
     link(href='/css/boilerplate.min.css', rel='stylesheet')
     title Express boilerplate
 
 mixin body()
   body
     block
+    script(src='https://code.jquery.com/jquery-3.3.1.min.js', integrity='sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=', crossorigin='anonymous')
+    script(src='/js/aj_logic.min.js', type='text/javascript')
```

Comment:
This should be, by default, `/js/boilerplate.min.js` as that is what the build logic will output.

Reply:
OK
Now script(src='/js/boilerplate.min.js', type='text/javascript')